packages/esm-patient-tests-app/src/test-results/panel-view/panel-view.component.tsx

```diff
@@ -96,13 +98,19 @@ const PanelView: React.FC<PanelViewProps> = ({ expanded, testUuid, basePath, typ
           )}
         </div>
         {activePanel ? (
-          <Overlay close={() => setActivePanel(null)} headerText={activePanel?.name}>
-            <PanelTimelineComponent groupedObservations={groupedObservations} activePanel={activePanel} />
-          </Overlay>
-        ) : null}
-        {trendlineView ? (
-          <Overlay close={navigateBackFromTrendlineView} headerText={activePanel?.name}>
-            <Trendline patientUuid={patientUuid} conceptUuid={testUuid} basePath={basePath} />
+          <Overlay close={() => setActivePanel(null)} headerText={activePanel?.key}>
+            <div className={styles.overlay}>
+              <TimelineDataGroup
+                patientUuid={patientUuid}
+                parent={activeTimelineParent}
+                subRows={activeTimelineData}
+                panelName={activePanel.key}
+                groupNumber={1}
+                setPanelName={() => {}}
+                xScroll={0}
+                setXScroll={() => {}}
```

Comment:
I don't know what these arguments do or are for but it seems to work like this. I suspect that there may be a bug if there are enough results in the timeline to be scrollable. My test data didn't have that. But anyway, definitely a lot less buggy than what was there!